Makefile

```diff
@@ -102,4 +103,3 @@ help:
 
 .PHONY: all clean deploy ensure-juju-env ensure-juju-test ftest help \
     jujutest lint setup sysdeps test unittest
```

Comment:
Not relevant for this branch, but we really need to move these .PHONY's to their corresponding targets.